tests/unit/test_alerts.py

```diff
@@ -160,7 +160,7 @@ def test_forwards_prometheus_rules(self):
         rules = json.loads(data["alert_rules"])
 
         for group in rules["groups"]:
-            if group["name"].endswith("provider-tester_alerts"):
+            if group["name"].endswith("provider-tester_alerts_alerts"):
```

Comment:
Is this a bug? I think the group should not end with "_alerts_alerts".

Reply:
mmm maybe it's a breaking change in the lib... I'll check it

Reply:
Hi @rbarry82 !

I found a bug in the regex changed [here](https://github.com/canonical/prometheus-k8s-operator/commit/d5d034182188f95b55990a6a6195c0656cdb3f50#diff-a15ebc3aa16d0affd9ee815751bc53054676385cd1f73bfc36bc3ad10b1c0da3R285)

With this change, the name of the alert group change from:

`lma_f2c1b2a6-e006-11eb-ba80-0242ac130004_provider-tester_alerts`

to:

`lma_12345678_grafana-agent-k8s_lma_f2c1b2a6-e006-11eb-ba80-0242ac130004_provider-tester_alerts_alerts`

because of [this piece of code](https://github.com/canonical/prometheus-k8s-operator/blob/main/lib/charms/prometheus_k8s/v0/prometheus_remote_write.py#L232):

```python
                if not self._is_already_modified(alert_group["name"]):
                    # update group name with topology and sub-path
                    alert_group["name"] = self._group_name(
                        str(root_path),
                        str(file_path),
                        alert_group["name"],
                    )
```

It is duplicating the final `_alerts` in the name.

should we revert the regex or the use case changed?